src/tracker-component.js

```diff
@@ -381,12 +380,13 @@ export const Tracker = (config? : Config = defaultTrackerConfig) => {
         }
     };
     const emptyTrackers = {
-        addToCart:      (data : CartData) => doNoop(), // eslint-disable-line no-unused-vars
-        setCart:        (data : CartData) => doNoop(), // eslint-disable-line no-unused-vars
-        removeFromCart: (data : RemoveCartData) => doNoop(), // eslint-disable-line no-unused-vars
-        purchase:       (data : PurchaseData) => doNoop(), // eslint-disable-line no-unused-vars
-        setUser:        (data : UserData) => doNoop(), // eslint-disable-line no-unused-vars
-        setPropertyId:  (id : string) => doNoop(), // eslint-disable-line no-unused-vars
+        addToCart:      (data : CartData) => doNoop(),
+        setCart:        (data : CartData) => doNoop(),
+        removeFromCart: (data : RemoveCartData) => doNoop(),
+        purchase:       (data : PurchaseData) => doNoop(),
+        setUser:        (data : UserData) => doNoop(),
+        cancelCart: (data : CancelCartData) => doNoop(),
```

Comment:
did lint complain about this?

Reply:
No. I can change it to match the others though.